.github/workflows/update-managed-batch-parser.yml

```diff
@@ -0,0 +1,142 @@
+name: Update ManagedBatchParser
+
+on:
+  schedule:
+  - cron: "50 11 * * 0" # 11:50 am UTC, every Sunday
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  get_versions:
+    name: Check current and latest ManagedBatchParser versions
+    runs-on: ubuntu-latest
+
+    outputs:
+      current: ${{ steps.get_current.outputs.current }}
+      latest: ${{ steps.output_jq.outputs.latest }}
+      download_url: ${{ steps.output_jq.outputs.url }}
+
+    steps:
+    - id: create_dirs
+      name: Create working directories
+      run: |
+        mkdir repo
+        mkdir working
+
+    - id: checkout
+      uses: actions/checkout@v4
+      with:
+        path: repo
+        
+    - id: get_current
+      name: Get currently-used version from version.txt
+      working-directory: repo
+      run: echo "current=$(cat src/DacpacTool/Microsoft.SqlTools.ManagedBatchParser/version.txt)" >> "$GITHUB_OUTPUT"
+
+      # Get sqltoolsservice release metadata,
+      # store in a file for later processing using JQ
+    - id: curl_sqltoolsservice
+      name: Get latest release metadata for sqltoolsservice
+      working-directory: working
+      run: curl https://api.github.com/repos/microsoft/sqltoolsservice/releases/latest | tee curl.txt
```

Comment:
Do we need to worry about any rate limiting issues here? I guess not since we're running this once a week, I guess I'm just curious ;).

Reply:
Per [these docs](https://docs.github.com/en/rest/using-the-rest-api/rate-limits-for-the-rest-api?apiVersion=2022-11-28), "The primary rate limit for unauthenticated requests is 60 requests per hour". :slightly_smiling_face: